tests/common.go

```diff
@@ -10346,3 +10350,37 @@ func GetNodeForGivenVolumeName(volName string) (*node.Node, error) {
 
 	return nil, fmt.Errorf("no attached node found for vol [%s]", volName)
 }
+
+func installGrafana(namespace string) {
```

Comment:
You dont want to return error in case the installation fails?

Reply:
I don't want block the runs as it will impact our tests.